### ISSUE TITLE
[Bug]: Install via chocolatey fails

### ISSUE THREAD

**Issue by annabunches** [ciderapp/Cider#1490]
### Describe the Bug

When trying to install via chocolatey, the following error occurs:

```
Chocolatey v1.1.0
Installing the following packages:
cider
By installing, you accept licenses for the packages.
Progress: Downloading cider 1.5.7... 100%

cider v1.5.7 [Approved]
cider package files install completed. Performing other installation steps.
The package cider wants to run 'chocolateyInstall.ps1'.
Note: If you don't run this script, the installation will fail.
Note: To confirm automatically next time, use '-y' or consider:
choco feature enable -n allowGlobalConfirmation
Do you want to run the script?([Y]es/[A]ll - yes to all/[N]o/[P]rint): A

Attempt to get headers for https://github.com/ciderapp/cider-releases/releases/download/v1.5.7-beta.2/Cider-Setup-1.5.7-beta.2.exe failed.
  The remote file either doesn't exist, is unauthorized, or is forbidden for url 'https://github.com/ciderapp/cider-releases/releases/download/v1.5.7-beta.2/Cider-Setup-1.5.7-beta.2.exe'. Exception calling "GetResponse" with "0" argument(s): "The remote server returned an error: (404) Not Found."
Downloading cider
  from 'https://github.com/ciderapp/cider-releases/releases/download/v1.5.7-beta.2/Cider-Setup-1.5.7-beta.2.exe'
ERROR: The remote file either doesn't exist, is unauthorized, or is forbidden for url 'https://github.com/ciderapp/cider-releases/releases/download/v1.5.7-beta.2/Cider-Setup-1.5.7-beta.2.exe'. Exception calling "GetResponse" with "0" argument(s): "The remote server returned an error: (404) Not Found."
This package is likely not broken for licensed users - see https://docs.chocolatey.org/en-us/features/private-cdn.
The install of cider was NOT successful.
Error while running 'C:\ProgramData\chocolatey\lib\cider\tools\chocolateyInstall.ps1'.
 See log for details.

Chocolatey installed 0/1 packages. 1 packages failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).

Failures
 - cider (exited 404) - Error while running 'C:\ProgramData\chocolatey\lib\cider\tools\chocolateyInstall.ps1'.
 See log for details.
```

tl;dr it's pointing to a nonexistent URL.

### Steps to Reproduce

1. Install chocolatey on Windows.
2. Run `choco install cider`.
3. Observe failure

### Environment Information

- OS: Windows 10
- Desktop Environment: default
- App Version: 1.5.7
- Commit ID: n/a

### How did you download the software?

Chocolatey

### Code of Conduct

- [X] I agree to follow this Cider's [Code of Conduct](https://github.com/ciderapp/Cider/blob/main/CODE_OF_CONDUCT.md)

### Disclaimer

- [X] I acknowledge the [Support Disclaimer](https://docs.cider.sh/support/disclaimer)

**Comment by coredev-uk** [comment]
As far as I know, we do not maintain the chocolatey package. Please contact the [maintainer](https://community.chocolatey.org/profiles/Xav83) with this information.
